src/odb/src/lefin/lefin.cpp

```diff
@@ -694,7 +694,7 @@ void lefin::layer(lefiLayer* layer)
       } else
         supported = false;
     } else if (type.getValue() == dbTechLayerType::CUT) {
-      if (!strcmp(layer->propName(iii), "LEF58_SPACING")) {
+      if (!strcmp(layer->propName(iii), "LEF58_SPACING") | !strcmp(layer->propName(iii), "LEF57_SPACING")) {
```

Comment:
Even though bitwise operator `|` is technically correct in this case it's much preferable to use the `||` logical operator when dealing with truth values. Please change to the logical operator `||`

Reply:
Done